src/appengine/handlers/crash_stats.py

```diff
@@ -131,11 +131,7 @@ def query_testcase(project_name, crash_type, crash_state, security_flag,
       data_types.Testcase.open == is_open,
       ndb_utils.is_false(data_types.Testcase.is_a_duplicate_flag)).order(
           -data_types.Testcase.timestamp).iter(
-              limit=1,
-              projection=[
-                  data_types.Testcase.bug_information,
```

Comment:
Is this really needed ? i prefer this style reference via doing typos in strings.

Reply:
This is https://github.com/googleapis/python-ndb/issues/292.

Reply:
Sorry meant https://github.com/googleapis/python-ndb/issues/295